manifests/init.pp

```diff
@@ -64,6 +67,7 @@
 
   $package_names = $katello::params::package_names,
   $enable_ostree = $katello::params::enable_ostree,
+  $max_keep_alive = $katello::params::max_keep_alive,
```

Comment:
The foreman module seems to use it, but i don't see it used much in the katello module.  Went ahead and added it.